modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.scss

```diff
@@ -22,8 +22,13 @@ hr {
 		font-weight: 600;
 		line-height: 1.3rem;
 		overflow: hidden;
+		padding-right: 4rem;
 		text-overflow: ellipsis;
 		white-space: nowrap;
 		width: 10rem;
+
+		&--danger {
+			color: #da1414;
+		}
```

Comment:
could you please also change this to the variable defined by frontend infra `$danger` as stated before?

Reply:
Done 😄